frontend/src/components/gmp/PenDisplay.vue

```diff
@@ -68,22 +76,25 @@ export default {
       copyText.select();
       copyText.setSelectionRange(0, 99999); /*For mobile devices*/
 
-      document.execCommand('copy');
-      this.clipboard = true;
-    } 
+      return navigator.clipboard.writeText(copyText.value)
+        .then(() => { this.clipboard = true; })
+        .catch(() => {
+          console.error('Something went wrong while copying text.'); // TODO: Re-work to use error snackbar.
```

Comment:
TODO?

Reply:
Turns out this file isn't even used.  I'm pruning it.